terraform/monitoring/lambda_end_to_end_bag_test.tf

```diff
@@ -37,3 +37,31 @@ module "end_to_end_bag_tester_prod" {
 
   tags = local.default_tags
 }
+
+# Allow the CI agent running in BuildKite to trigger the Lambda after
+# deploying new images to staging.
+# See https://github.com/wellcomecollection/platform/issues/4819
+resource "aws_lambda_permission" "allow_platform_to_trigger_lambda" {
```

Comment:
Does it make sense to describe these CI permissions where the CI role is created in platform-infra (I think there are other specific permissions there).

Reply:
I thought about it; it's not really obvious to me that one makes more sense than the other.

I picked here just because it means one PR rather than two.